mii/grpc_related/restful_gateway.py

```diff
@@ -0,0 +1,62 @@
+import time
+import threading
+import mii
+from flask import Flask, request
+from flask_restful import Resource, Api
+from werkzeug.serving import make_server
+from mii.constants import RESTFUL_GATEWAY_SHUTDOWN_TIMEOUT, RESTFUL_API_PATH
+from google.protobuf.json_format import MessageToJson
+
+
+def shutdown(thread):
+    time.sleep(RESTFUL_GATEWAY_SHUTDOWN_TIMEOUT)
```

Comment:
What purpose does this `sleep` serve when we terminate?

Reply:
The server would fail to return the response of `terminate` request if the server process finishes soon. This sleep delays the termination of the server process.